tutorials/vscodeJavaSetup.md

```diff
@@ -0,0 +1,182 @@
+{% set title="VS Code - Setting up a Java project" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+This tutorial will guide you through creating and managing Java projects in VS Code.
+
+<box type="info" seamless>
+
+This tutorial is adapted from the [official VS Code Java Tutorial](https://code.visualstudio.com/docs/java/java-tutorial).
+</box>
+
+## Install the Extension Pack for Java
+
+To develop Java applications in VS Code, you need to install the appropriate extensions that provide language support, debugging, and other essential features. The easiest way to get started with Java development in VS Code is to install the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes all the essential extensions you need.
+
+### Installation steps
+
+1. **Open the Extensions View**: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+X` | {{ icon_apple }} `Cmd+Shift+X`
+2. **Search for** `Extension Pack for Java`
+3. **Click** `Install` on the extension by Microsoft
+
+### What's Included
+
+The [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) includes these essential tools for Java development:
+* **Language Support for Java** - Syntax highlighting, IntelliSense, and error detection
+* **Debugger for Java** - Set breakpoints and step through your code
+* **Test Runner for Java** - Run and debug JUnit tests
+* **Maven for Java** - Support for Maven projects
+* **Gradle for Java** - Support for Gradle projects
+* **Project Manager for Java** - Manage Java projects and dependencies
+* **IntelliCode** - AI-assisted code completion
+
+<box type="tip" seamless>
+
+**Quick tip:** After installation, open the Command Palette: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+P` | {{ icon_apple }} `Cmd+Shift+P`, type `Java: Tips for Beginners` and open it for helpful information to get started.
```

Comment:
Give screenshot? Apparently, it expects a `>` at the start.
![image](https://github.com/user-attachments/assets/cf5fc7bb-1940-4954-b813-a6a859d71830)